spec/factories/terms_of_use_agreements.rb

```diff
@@ -0,0 +1,9 @@
+# frozen_string_literal: true
+
+FactoryBot.define do
+  factory :terms_of_use_agreement do
+    association :user_account
+    agreement_version { 'V1' }
```

Comment:
Not a huge deal since this is a spec but I think SuS wanted version 3 and on the FE we would display version 1